react-first-app/src/app/components/App/Square/index.js

```diff
@@ -0,0 +1,15 @@
+import React, { Component } from 'react';
+
+import logo from '../../../../logo.svg';
+
+import style from './styles.css';
+
+function Square(props) {
```

Comment:
Instead of receiving `props`, we like to destructure them to use only the variables we want ant to specify better what it receives:
`function Square({ value, onClick })`